lib/WasmReader/WasmByteCodeGenerator.cpp

```diff
@@ -549,6 +549,9 @@ WasmBytecodeGenerator::EmitExpr(WasmOp op)
         info = EmitInfo(tempReg, WasmTypes::I32);
         }
         break;
+    case wbUnreachable:
+        m_writer.EmptyAsm(Js::OpCodeAsmJs::Unreachable_Void);
```

Comment:
I mean, add this after the writer.
The info tells the bytecode generator that we have entered in an `unreachable` state after this opcode.
It changes slightly how we (will) handle bytecode generation.
Basically, we won't emit bytecode when we are in `unreachable` code, but we still have to validate it.